data/uhc_pack/functions/pre_generation/complete.mcfunction

```diff
@@ -1,9 +1,10 @@
-
 scoreboard objectives setdisplay sidebar uhcLobby
-scoreboard players reset * uhcPG
 
-tp @a[tag=pre_generator] @e[type=minecraft:armor_stand,tag=lobbycenter,limit=1]
-tag @a[tag=pre_generator] remove pre_generator
+scoreboard players reset * uhcPGSet
+
+execute in minecraft:the_nether run forceload remove all
```

Comment:
I would prefer we don't take this shortcut. If for whatever reason an admin has forceloaded a chunk, I would like if we maintain this

Reply:
Reworking this now! I'll make sure to also keep this into account when the armor stand goes to the chunk to load it.